app/models/immunisation_import_row.rb

```diff
@@ -89,8 +89,8 @@ def to_vaccination_record
     if vaccination_record.persisted?
       vaccination_record.stage_changes(
         batch_id: batch.id,
-        delivery_method:,
-        delivery_site:
+        delivery_method: delivery_method.to_s,
+        delivery_site: delivery_site.to_s
```

Comment:
I wonder if it makes sense to change the `delivery_method` and `delivery_site` methods themselves to return the value as a string so it's the same everywhere.

Since the model instance returns them as strings, this might avoid any similar issues in the future.

Reply:
Thought about doing that instead 👍